CHANGELOG.md

```diff
@@ -1,4 +1,44 @@
-# Changelog
+# CHANGELOG
+
+## 1.25.0 (2022-04-11)
+
+MLflow 1.25.0 includes several major features and improvements:
+
+Features:
+
+- [Tracking] Introduce a new fluent API `mlflow.last_active_run()` that provides the most recent fluent active run (#5584, @MarkYHZhang)
+- [Tracking] Add `experiment_names` argument to the `mlflow.search_runs()` API to support searching runs by experiment names (#5564, @r3stl355)
+- [Tracking] Add a `description` parameter to `mlflow.start_run()` (#5534, @dogeplusplus)
+- [Tracking] Add `log_every_n_step` parameter to `mlflow.pytorch.autolog()` to control metric logging frequency (#5516, @adamreeve)
+- [Tracking] Log `pyspark.ml.param.Params` values as MLflow parameters during PySpark autologging (#5481, @serena-ruan)
+- [Tracking] Add support for  `pyspark.ml.Transformer`s to PySpark autologging (#5466, @serena-ruan)
+- [Tracking] Add input example and signature autologging for Keras models (#5461, @bali0019)
+- [Models] Add support for logging models to preexisting destination directories (#5572, @akshaya-a)
+- [Models] Introduce `mlflow.diviner` flavor for large-scale [time series forecasting](https://databricks-diviner.readthedocs.io/en/latest/?badge=latest) (#5553, @BenWilson2)
+- [Models] Add `mlflow_version` field to MLModel specification (#5515, #5576, @r3stl355)
+- [Models] Add `pyfunc.get_model_dependencies()` API to retrieve reproducible environment specifications for MLflow Models with the pyfunc flavor (#5503, @WeichenXu123)
+- [Models] Add `code_paths` argument to all model flavors to support packaging custom module code with MLflow Models (#5448, @stevenchen-db)
+- [Models] Support creating custom artifacts when evaluating models with `mlflow.evaluate()` (#5405, #5476 @MarkYHZhang)
+- [Scoring / Projects] Introduce `--env-manager` configuration for specifying environment restoration tools (e.g. `conda`) Deprecate `--no-conda` (#5567, @harupy)
+- [Scoring] Support restoring model dependencies in `mlflow.pyfunc.spark_udf()` to ensure accurate predictions (#5487, #5561, @WeichenXu123)
+- [Scoring] Add support for `numpy.ndarray` type inputs to `tensorflow` predict function (#5545, @WeichenXu123)
+- [Scoring] Support deployment of MLflow Models to Sagemaker Serverless (#5610, @matthewmayo)
+- [UI] Add MLflow version to header beneath logo (#5504, @adamreeve)
+- [Artifacts] Introduce a `mlflow.artifacts.download_artifacts()` API mirroring the functionality of the `mlflow artifacts download` CLI (#5585, @dbczumar)
```

Comment:
Maybe it is just me, but when I see `mlflow.artifacts.download_artifacts()`, I get an impression that `download_artifacts` can be called without arugments.

Reply:
I don't get that impression, but I'll defer to @BenWilson2 here :)

Reply:
btw what does `xxx()` indicate? Does this indicate `xxx` a callable object?

Reply:
Looks like pandas uses this format:

https://pandas.pydata.org/docs/whatsnew/v1.4.1.html

Reply:
> btw what does `xxx()` indicate? Does it indicate `xxx` is a function?

Yes

Reply:
I see, I'm personally not a big fan of this format because `xxx()` looks like a function *call*, not a function, but I'm fine because other packages also use this format.

Reply:
yeah I've always used the inclusion of () in doc strings to indicate a method or function (callable) and without it as a module name. This isn't to say that it's correct and I have no particular conviction one way or another (so long as it's kept standardized :) ).